src/integrationTest/java/org/opensearch/test/framework/matcher/GetSettingsResponseContainsIndicesMatcher.java

```diff
@@ -35,15 +32,12 @@ class GetSettingsResponseContainsIndicesMatcher extends TypeSafeDiagnosingMatche
 	@Override
 	protected boolean matchesSafely(GetSettingsResponse response, Description mismatchDescription) {
 
-		final Map<String, Settings> indexToSettings = new HashMap<>();
-		for (ObjectObjectCursor<String, Settings> cursor : response.getIndexToSettings()) {
-			indexToSettings.put(cursor.key, cursor.value);
-		}
+		final ImmutableOpenMap<String, Settings> indexToSettings = response.getIndexToSettings();
 		for (String index : expectedIndices) {
 			if (!indexToSettings.containsKey(index)) {
 				mismatchDescription
 						.appendText("Response contains settings of indices: ")
-						.appendValue(indexToSettings.keySet());
+						.appendValue(indexToSettings.keys());
```

Comment:
@scrawfor99 Sorry I had to push a small change and now CI is running again. I had this as `keysIt` when it really should be the key set here.